packages/frontend/src/features/semanticViewer/components/SemanticViewerVizConfig/CartesianVizStyling.tsx

```diff
@@ -0,0 +1,5 @@
+import { CartesianChartStyling } from '../../../../components/DataViz/config/CartesianChartStyling';
+
+// NOTE: This file is included here, but the styling is shared from the
+// data viz folder for now. If the styling diverges, we can add it here.
+export { CartesianChartStyling as CartesianVizStyling };
```

Comment:
The styling form is shared. It's called out that way in the eraser 'spec' and it doesnt currently have anything specific to semantic viewer, so I kept in in common for now. I put this file here to make it clear that we are generally not sharing the configs, but we could do this a different way if anyone has an opinion.